### PR TITLE
incorporate code review comments

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: src/style/themes/default/antd/tootltip.less [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.@{tooltip-prefix-cls} {
+  &-inner {
+    color: @tooltip-color;
+  }
+}
```
Comment (francisl) [inline src/style/themes/default/antd/tootltip.less:5]: why is this necessary? since you already settle value of the variable, the theme should pick it up.

Comment (evans-g-crsj) [reply src/style/themes/default/antd/tootltip.less]: good point!

**Review by francisl — APPROVED** [review]
